vnext/Desktop/Modules/WebSocketModule.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// clang-format off
 #include "pch.h"
```

Comment:
remove

Reply:
Clang formatting makes the heavily lambda-based code in this unit hard to both maintain and read.

Is it OK to leave an exception?

Reply:
My personal opinion is that we should almost never disable formatting for an entire file.

If we have problem areas with our rules, I think we should consider changing the rules. Some of this I think ends up being stylistic/taste though. On my old team there was a holy war over people wanting to keep Whitesmiths style braces (look it up, it's God awful) over Allman, which they considered hard to read. 

Occasional sections can be really problematic and we can turn things off there. If we generalize engineers turning off formatting in files though where they don't like part of the result, we end up with no consistent formatting anywhere.

Reply:
I believe you have mentioned that before in this review.
Before, I wasn't aware it could be disabled per-section.

I'll apply in a scoped way.